crates/rooch-types/src/transaction/mod.rs

```diff
@@ -170,7 +171,7 @@ pub struct TransactionInfo {
     /// The vm status. If it is not `Executed`, this will provide the general error class. Execution
     /// failures and Move abort's receive more detailed information. But other errors are generally
     /// categorized with no status code or other information
-    pub status: KeptVMStatus,
+    pub status: VmStatusExplainView,
```

Comment:
the View type only for the JSON RPC output, the transaction info should keep the original Status

Reply:
Okay, it has been fixed.